schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
This should be a separate PR

Reply:
would be nice, but:

z-schema is stuck at draft-4
ajv doesn't work for draft-4
so, I'd have to revert back to z-schema for these tests, and then in some later PR bump the schema draft version and then validation framework.

All possible, but it creates multiple dependent pull requests and would take a lot more time than I have to work on this project at the moment.

Reply:
so okay to mark this as resolved?